docs/release-notes/8.0/8.0.34-29.0.md

```diff
@@ -0,0 +1,29 @@
+# Percona XtraBackup 8.0.34-29 (2023-08-07)
+
+[Percona XtraBackup (PXB)](https://www.percona.com/software/mysql-database/percona-xtrabackup) is a 100% open source backup solution for all versions of Percona Server for MySQL and MySQL® that performs online non-blocking, tightly compressed, highly secure full backups on transactional systems. Maintain fully available applications during planned maintenance windows with Percona XtraBackup.
+
+## Release highlights
+
+This release merges the MySQL 8.0.34 code base. Percona has implemented a two-stage release process for each version. The first release primarily ensures compatibility with the latest MySQL version, to help those customers who need an updated version of Percona XtraBackup as soon as possible. The second release contains additional bug fixes and any improvements or new features.
+
+The [ZSTD compression method]() is moved to [General Availability](..//..//glossary.md#general-availability-ga) and becames default compression method for `--compress` option.
```

Comment:
the default compression method for the `--compress` option.

Reply:
done